controllers/users/users.js

```diff
@@ -3,6 +3,7 @@ const bcrypt = require("bcryptjs");
 const jwt = require("jsonwebtoken");
 const config = require("config");
 const User = require("../../models/User");
+const { SE, EAE, UR } = require("../messages");
```

Comment:
```
const messages = require('../messages')
```

It won't be better?
And in every messages changing to -> message.USER_DOESNT_EXIST